Source/MonoGame.Extended/BitmapFonts/BitmapFont.cs

```diff
@@ -44,28 +44,48 @@ internal static IEnumerable<int> GetUnicodeCodePoints(string s)
 
         public Size GetSize(string text)
         {
+            var totalWidth = 0;
             var width = 0;
+            var totalHeight = 0;
             var height = 0;
+
+            const int newlineCodePoint = 10;
             var codePoints = GetUnicodeCodePoints(text).ToArray();
 
             for (int i = 0, l = codePoints.Length; i < l; i++)
             {
                 BitmapFontRegion fontRegion;
                 var c = codePoints[i];
+                var d = c;
```

Comment:
Will do, was just trying to follow the "convention" in place thus far. Do you have any problem with me commenting as well?